notebooks/05_operationalize/aks_locust_load_test.ipynb

```diff
@@ -114,34 +114,26 @@
    "outputs": [],
```

Comment:
was this change intentional?

Reply:
Which change is this comment commented on?

Reply:
this is regarding the locust load testing notebook

Reply:
I made some small changes with where the path to the config file could be found. The big change of deleting the 2 cells that verify the service was because they threw errors when I ran them with our two endpoints.

Reply:
i don't think we need to capture the application of this notebook for load testing the app, we should leave it as it was (or provide an update on how to set up the config file if that part led to confusion).

Reply:
Ah makes sense, I'll remove the notebook from here